scipy/optimize/_differentialevolution.py

```diff
@@ -294,6 +295,53 @@ def differential_evolution(func, bounds, args=(), strategy='best1bin',
     iteration, rather than multiple times for all the population members; the
     interpreter overhead is reduced.
 
+    An outline of the strategies and their stengths and weaknesses is given by 
```

Comment:
`An outline of the other strategies and their strengths and weaknesses is given by ...`